src/hooks/cart.js

```diff
@@ -13,8 +13,13 @@ export default class extends BaseHooks {
     }
 
     itemAdd() {
-        this.$body.on('submit', '[data-cart-item-add]', (event) => {
-            this.emit('cart-item-add', event, event.target);
-        });
+        const elements = document.querySelectorAll('[data-cart-item-add]');
+        if (elements.length) {
```

Comment:
No need for this `if` -- the loop will simply not iterate if `elements.length` is zero. Same for similar instances below.

Reply:
fixed.